sycophancy_analysis/dashboard/app.py

```
@@ -0,0 +1,894 @@
+            print(f"Warning: {PROMPT_BATTERY_FILE} not found; falling back to programmatic battery build")
+            try:
+                # Build in-memory prompt battery from the code if file is missing
+                from sycophancy_analysis.data.prompts import build_sycophancy_battery
+                df = build_sycophancy_battery()
+                items = df.to_dict('records')
+                # Best-effort: persist for the dashboard to serve next time
+                os.makedirs(os.path.dirname(PROMPT_BATTERY_FILE), exist_ok=True)
+                with open(PROMPT_BATTERY_FILE, 'w', encoding='utf-8') as f:
+                    json.dump(items, f, ensure_ascii=False, indent=2)
+                print(f"Rebuilt and saved prompt battery to {PROMPT_BATTERY_FILE}")
+                return items
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)